Client/mods/deathmatch/logic/CRadarMap.cpp

```diff
@@ -114,6 +122,24 @@ CRadarMap::~CRadarMap()
     // Don't need to delete the help texts as those are destroyed by the display manager
 }
 
+void CRadarMap::SetMapImage(std::uint32_t imageIndex)
+{
+    std::uint32_t width, height;
+    if (imageIndex < RADAR_IMAGE_SIZES.size())
+        width = height = RADAR_IMAGE_SIZES[imageIndex];
+    else
+        width = height = RADAR_IMAGE_SIZES[0];
```

Comment:
Is this check necessary? Can be `imageIndex` invalid?

Reply:
imageIndex indeed can't be invalid because the value is clamped by the settings script.
I can remove it, it's just a failsafe in case something goes wrong with client setting var